<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>845</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>190</red>
        <green>190</green>
        <blue>190</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Human Classifier</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0.269663 rgba(85, 170, 127, 255), stop:1 rgba(255, 255, 255, 255))
}

QLabel#label {
	min-height: 5em
}

QLabel#label_2, QLabel#rounds {
	min-height: 5em;
	margin: 5% 10%;
}

QLabel#p1label {
	min-height: 4em;
	border-style: solid;
	border-width: 2px;
	border-radius: 6px;
	border-color: blue
}

QLabel#p1Points {
	min-height: 9em;
	background-color: lightblue;
	border-style: solid;
	border-width: 2px;
	border-radius: 6px;
	border-color: blue
}

QLabel#p2label {
	border-style: solid;
	border-width: 2px;
	border-radius: 6px;
	border-color: red
}

QLabel#p2Points {
	background-color: rgb(255, 125, 145);
	border-style: solid;
	border-width: 2px;
	border-radius: 6px;
	border-color: red
}

QPushButton {
	background-color: lightgrey;
	border-style: outset;
	border-width: 2px;
	border-radius: 6px;
	border-color: grey;
  	min-height: 20px;
	padding: 4px;
}

QPushButton#pb_close{
	margin: 7% 40%
}

QPushButton#pb_reset{
	margin: 7% 40%
}

QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0.006, y1:0.915182, x2:0, y2:0, stop:0.0798991 rgba(195,195,195,255), stop:0.330322 rgba(203,203,203,255), stop:1 rgba (255,255,255,255));
	border-color: rgb(45, 45, 45);
	border-style: inset;
}

QComboBox {
    border: 1px solid gray;
    border-radius: 6px;
    padding: 1px 18px 1px 3px;
	min-height: 2em;
    min-width: 6em;
}

QComboBox:editable {
    background: white;
}

QComboBox:!editable, QComboBox::drop-down:editable {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                 stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
}

/* QComboBox gets the &quot;on&quot; state when the popup is open */
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
                                stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);
}

QComboBox:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {
    image: url(d:/Schule/5BHIT/SEW/RockPaperScissors/icons/arrow.png);
}

QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="palette">
    <palette>
     <active/>
     <inactive/>
     <disabled/>
    </palette>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="p1Choice">
        <property name="styleSheet">
         <string notr="true">text-align: center</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pb_none">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_natural">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Natural</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_sea_ice">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Sea Ice</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_agricultural">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Agricultural</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_water">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Water</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_cloud">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Cloud</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_river">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>River</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_snow">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Snow</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_mountain">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Mountain</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_mountain</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>596</x>
     <y>509</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
